### PR TITLE
Added Swastika I/O Core CMS.

### PR DESCRIPTION
**Pull request by Smilefounder**

### CODE REVIEW

**Comment by Smilefounder** [comment]
Hi @thangchung , please feel free to let me know if there is anything wrong with my pull request. I will fix the issues accordingly.

**Comment by ngohungphuc** [comment]
Please resolve conflict

**Comment by Smilefounder** [comment]
Hi @ngohungphuc resolved, please help check again. Thanks.

File: README.md [hunk 1]
```diff
@@ -217,10 +217,10 @@ Thanks to all [contributors](https://github.com/thangchung/awesome-dotnet-core/g
 * [Platformus](https://github.com/Platformus) - Free, open source and cross-platform CMS based on ASP.NET Core 1.0 and ExtCore framework.
 * [SimpleContent](https://github.com/joeaudette/cloudscribe.SimpleContent) - Simple, yet flexible content and blog engine for ASP.NET Core that can work with or without a database.
 * [Squidex](https://github.com/Squidex/squidex) - Headless CMS, based on MongoDB, CQRS and Event Sourcing.
+* [Swastika I/O Core CMS](https://github.com/Swastika-IO/Swastika-IO-Core) - Open source ASP.NET Core 2.x CMS. It currently supports MS SQL and planned to implement MSSQL, SQLite in the near future. It has many built-in features out of the box like multilanguage support, theme, template...
```
Comment (MaceWindu) [inline README.md]: > supports MS SQL and planned to implement MSSQL
should it be MySQL in plans?

Comment (thangchung) [reply README.md]: @Smilefounder Please help to clarify this one!

**Pull request merged**